kyaml/internal/forked/update-go-yaml.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
+
+set -euo pipefail
+
+if [ "$#" -ne 1 ]; then
+    echo "Usage: $0 \$GOYAML_V3_SHA"
+    exit 1
+fi
+
+if [ "$(git branch --show-current)" == "master" ]; then
+  echo "You must be on a branch to use this script."
+  exit 1
+fi
+
+blue=$(tput setaf 4)
+normal=$(tput sgr0)
+
+function explain() {
+    printf "\n\n%s\n" "${blue}$1${normal}"
+}
+
+# This should be the version of go-yaml v3 used by kubectl
+# In the original fork, this is 496545a6307b2a7d7a710fd516e5e16e8ab62dbc
+export GOYAML_SHA=$1
+export GOYAML_REF="goyaml-$GOYAML_SHA"
+
+# This is the sha of the commit in https://github.com/go-yaml/yaml/pull/753
+export CHERRY_PICK_SHA=e965076f45946b0bda907e6c2316b4523207630e
```

Comment:
maybe a dumb question, but where did this sha come from?

Reply:
Your PR 😄 
This needs to be updated once your PR is finalized. Right now this is the revision before you made some naming changes yesterday.